### DISCUSSION TITLE
Animate from one plot to another

### DISCUSSION THREAD

**Discussion by giorgi-ghviniashvili** [observablehq/plot#434] [Ideas]
Is it possible to transform one plot to another using `d3-transition`? 

For example, in this [notebook](https://observablehq.com/@giorgi-ghviniashvili/exploring-observable-plot), is it possible to smoothly animate from `simple info-grid` to `faceted info-grid` and then to `jitter`?

**Comment by Fil** [comment]
There's nothing built in Plot 0.1 to do this type of stuff, but we can explore some possibilities… it's only two data-bound SVGs after all.

If you have the same elements (ie svg:circles) corresponding to the same data points (same cardinality), then it should be technically possible to animate from one to the other.

Note that the __data__ property attached to each element is the index of the data point in the mark. Usually, you'll see 0, 1, 2, 3… but if a mark is invalid (for example has an undefined *x*, *y* or *r*), then the svg element is absent.

Transitioning the axes might be possible too, but it depends on their types.

(PS: I feel that this should be a discussion rather than an issue.)

**Comment by giorgi-ghviniashvili** [comment]
@Fil thanks. I will try to animate it following to your idea. 
About the issue vs discussion, yes I agree that this should be a discussion.

**Comment by arky** [comment]
Cross posting the question I asked in Community forum. I think there is value in providing some kind of way to smooth transitions. This is available in some libraries such as datawrapper and flourish. 


https://talk.observablehq.com/t/animating-transitions-between-plots/6337
